overlays/haskell.nix

```diff
@@ -491,7 +491,7 @@ self: super: {
             happy = self.buildPackages.haskell-nix.bootstrap.packages.happy;
             hscolour = self.buildPackages.haskell-nix.bootstrap.packages.hscolour;
             ghc865 = self.buildPackages.haskell-nix.compiler.ghc865;
-            ghc882 = self.buildPackages.haskell-nix.compiler.ghc882;
```

Comment:
So we don't want to retain ghc882?

Reply:
I suppose we could.  Overhead CI of changes to things that trigger ghc rebulds `compiler/ghc/default.nix` will go up.  Not sure if that is really a problem as the should be infrequent.